charts/grocy/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+appVersion: v2.6.1
+description: ERP beyond your fridge - grocy is a web-based self-hosted groceries & household management solution for your home
+name: grocy
+version: 1.0.0
+keywords:
+  - grocy
+home: https://github.com/billimek/billimek-charts/tree/master/charts/grocy
+icon: https://github.com/grocy/grocy/raw/master/public/img/appicons/mstile-150x150.png
+sources:
+  - https://github.com/grocy/grocy
+maintainers:
+  - name: Ryan Holt
```

Comment:
Can you please make this your github username (`carpenike`)? The linter seems to only want github usernames.

Reply:
resolved by 537ff35